user-guide/01-introduction/1-1-quickstart.njk

```diff
@@ -0,0 +1,15 @@
+---
+title: Quickstart
+---
+
+<h1 id="section-1-1">Quickstart</h1>
+
+<p class="main-paragraph">You can start using Penpot right in your browser or installing it in a server of your own.</p>
+
+<h3>In the browser</h3>
+<p>To use Penpot online point your browser to designer.gravit.io and start designing. Use the latest Google Chrome or Mozilla Firefox for the best experience. We also provide specific support to WebKit (Safari / Epiphany).</p>
+<p>You simply go to <a href="https://penpot.app" target="_blank">penpot.app</a> and click on the Signup button. You will be asked to create an account. We only ask for an email. There are some authentication providers available, too.</p>
+
+<h3>In my own server</h3>
+<p>Current private Penpot instances are only requiring basic Docker knowledge.</p>
+<p>You can run your own Penpot server following these instructions: <a href=">https://help.penpot.app/developer-guide/configuration" target="_blank">help.penpot.app/developer-guide/configuration</a></p>
```

Comment:
For internal links, think it's better to use relative urls, and do not open a new window. In this case it would be, for example:

`<p>You can run your own Penpot server following <a href="/developer-guide/configuration"> these instructions</a>.</p>`